webpack.config.js

```diff
@@ -152,7 +152,7 @@ module.exports = env => {
       splitChunks: {
         cacheGroups: {
           commons: {
-            test: /[\\/]node_modules[\\/](react|react-dom|redux)[\\/]/,
+            test: /[\\/]node_modules[\\/](react|react-dom|redux|@patternfly*)[\\/]/,
```

Comment:
Good catch! This means that if you are building commons package it will include `@patternfly` in it and load it correctly in minified version. Otherwise the treeshaking could remove this package from build.